person.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'high_five'
+
+class Person
+  include HighFiveable
+
+  require_relative 'high_five'
+	attr_reader :age, :salary, :phase, :target_raise
+	attr_accessor :name
+
+	def initialize(options = {})
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+	end
+end
```

Comment:
Still something wonky going on with your indentation.  Double check that you've got sublime tabs set to 2 spaces: http://stackoverflow.com/questions/9474090/how-do-i-force-sublime-text-2-to-indent-two-spaces-per-tab